plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -176,6 +180,21 @@ struct FileIndex {
         line.addClass("loc");
 
         line.addChild("span", "1:").addClass("line_nr");
+        auto mut_data = "var g_muts_data = {};\n";
+        mut_data ~= "g_muts_data[-1] = {'kind' : null, 'status' : null, 'testCases' : null, 'orgText' : null, 'mutText' : null, 'meta' : null};\n";
+        alias sort_tcs_on_kills = (a, b) => db.getTestCaseInfo(a, kinds)
+            .killedMutants < db.getTestCaseInfo(b, kinds).killedMutants;
+
+        //Finds the index of a kind in a list of kinds
+        auto find_kind_index = function(const Mutation.Kind a, const Mutation.Kind[] lkinds) {
```

Comment:
Change it to a normal function definition as:
```
static auto findKindIndex(const Mutation.Kind a, const Mutation.Kind[] lkinds) {....}
```

Reply:
I would recommend changing the index lookup to include all members of Mutation.Kind.
Then the use of this function could be replaced with a simple conversion to ubyte.
Pseudo-code:
```
addChild(new RawSource(ctx.doc, format("const g_mut_kind_map = [%('%s',%)'];", 
[EnumMembers!(Mutation.Kind))));
....
// find_kind_Index replaced with this
m.mut.kind.to!ubyte
```

Reply:
This function was a result of the old mutant filter, which filtered on the enum "Mutant" and not "MutantKind". This allowed us to only add the mutants actually used in the testing. This function ensured that we got the correct index in the potentially shorter list. This can be removed.